py27/requirements-bin.txt

```diff
@@ -1,8 +1,8 @@
 anyjson-0.3.3-cp27-none-any.whl
-# Basemap will be part of an extra package
-#basemap-1.0.8-cp27-cp27mu-manylinux1_x86_64.whl
+basemap-1.0.8-cp27-cp27mu-manylinux1_x86_64.whl
```

Comment:
Keep basemap commented out (with the related comment), as we discussed yesterday